utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/util.scala

```diff
@@ -1,12 +1,18 @@
 package pl.touk.nussknacker.engine.util.functions
 
-import pl.touk.nussknacker.engine.api.Documentation
+import pl.touk.nussknacker.engine.api.{Documentation, HideToString, ParamName}
 
 import java.util.UUID
+import scala.jdk.CollectionConverters._
 
-object util {
+object util extends HideToString {
 
   @Documentation(description = "Generate unique identifier (https://en.wikipedia.org/wiki/Universally_unique_identifier)")
   def uuid: String = UUID.randomUUID().toString
 
+  // todo: can be removed once we have a better support for Java arrays in SpEL expressions, then we could simply use `"".split("regex")`
+  @Documentation(description = "Splits given text into a list of Strings using given regular expression")
```

Comment:
WDYT about adding notice, that empty trailing empty strings won't be discarded as in standard, java String.split?
Without this I see dangerous that someone can think that it is jus "".split("x", 0)

Reply:
Good idea. Done